src/index.js

```diff
@@ -34,7 +34,7 @@ export default class DynamicCdnWebpackPlugin {
 
     apply(compiler) {
         if (!this.disable) {
-            this.execute(compiler, {env: this.env});
+            this.execute(compiler, {env: this.env || compiler.options.mode || 'production'});
```

Comment:
Here how `compiler.options.mode` I believe should map:
- `none`: `development`
- `undefined`: `production`

production and development says the same

Reply:
probably moving the logic into its own function will make is nicer?